doc/classes/JSON.xml

```diff
@@ -53,7 +53,7 @@
 		<method name="get_parsed_text" qualifiers="const">
 			<return type="String" />
 			<description>
-				Return the text parsed by [method parse] as long as the function is instructed to keep it.
+				Return the text parsed by [method parse] (requires [param keep_text] to be specified to [method parse]).
```

Comment:
You cannot reference parameters from other functions this way, at most you'd have to use `[code]` tags

Reply:
```suggestion
				Return the text parsed by [method parse] (requires [code]keep_text[/code] to be passed to [method parse]).
```
Or similar